tests/e2e/helpers/params.js

```diff
@@ -1,7 +1,7 @@
 // @ts-check
 class TestParams {
   baseUrl = process.env.BASE_URL?.replace(/\/+$/, '') ?? 'http://127.0.0.1:8000'
-  ocaids = process.env.OCAIDS?.split(',') ?? [];
+  ocaids = process.env.OCAIDS?.split(',') ?? null;
```

Comment:
this was the issue where it was not receiving the `ocaids` to run tests on...

Reply:
🤦‍♂️ D'oh! I vaguely recall thinking "It's nice that the type of this is consistent now" 🤦‍♂️🤦‍♂️🤦‍♂️